ui/src/app/utils/access-utils.tsx

```diff
@@ -134,6 +135,14 @@ export const getAccessModuleConfig = (moduleName: AccessModule): AccessModuleCon
       refreshAction: async () => await profileApi().syncComplianceTrainingStatus(),
     })],
 
+    [AccessModule.CTCOMPLIANCETRAINING, () => ({
+      moduleName,
+      isEnabledInEnvironment: enableComplianceTraining,
+      DARTitleComponent: () => <div>Bypass {environment.displayTag} <AoU/> research Controlled Tier training </div>,
+      // externalSyncAction: async () => await profileApi().syncComplianceTrainingStatus(),
```

Comment:
please replace these with a comment indicating that these have not yet been implemented.